test/specs.expected

```diff
@@ -1106,7 +1106,8 @@ list-revdeps: debian-12-ocaml-4.14/amd64 opam-dev
     COPY --chown=1000:1000 . opam-repository/
     RUN opam repository set-url --strict default opam-repository/
     RUN opam update --depexts || true
-    RUN echo '@@@OUTPUT' && opam list -s --color=never --depends-on 'a.0.0.1' --coinstallable-with 'a.0.0.1' --all-versions --depopts && opam list -s --color=never --depends-on 'a.0.0.1' --coinstallable-with 'a.0.0.1' --all-versions --recursive && opam list -s --color=never --depends-on 'a.0.0.1' --coinstallable-with 'a.0.0.1' --all-versions --with-test --depopts && echo '@@@OUTPUT'
+    RUN opam pin add -k git -y opam-ci-check git+https://github.com/ocurrent/opam-repo-ci.git#live
```

Comment:
This step could get cached and in a Dockerfile we could use the `ADD` command to work around the caching:

```dockerfile
ADD https://api.github.com/repos/punchagan/opam-repo-ci/git/refs/heads/revdeps-exp version.json
```

but obuilder spec doesn't seem to support `ADD` and I'm not sure there's a nice way to do this pinning with commit hashes since we'd want to pin to the commit after the merge.

Reply:
We discussed three ideas in our co-working meeting:

- Extending obuilder with `add`
- Some way to ensure that previous actions in docker container would invalidate the cache when needed
- Deploying a docker image alongside the other deployment with `opam-ci-check` and using that as the base